### DISCUSSION TITLE
ffmpeg

### DISCUSSION THREAD

**Discussion by loveisglitchy** [Androz2091/discord-player#349] [General]
Could you attach "ffmpeg-static" and "@discordjs/opus" to the package.json so that we don't have to install it ourselves in the future and just jump straight into making the music client and commands?

I do see that opus is in the devDeps but if you put it in deps, maybe it'll install it for us as well. 

I am aware that different OS's require them a bit differently, so maybe make it a function to install the proper packages on the current OS?

**Comment by twlite** [comment]
# Why is `@discordjs/opus` devDependency?
Because you can select one of these `opus` engine: **[@discordjs/opus](https://npmjs.com/package/@discordjs/opus)**, **[node-opus](https://npmjs.com/package/node-opus)** and **[opusscript](https://npmjs.com/package/opusscript)**.

# Why is `ffmpeg-static` devDependency?
We don't wanna re-download `ffmpeg` if users have it installed on their system.
